enhancements/installer/ibmcloud-ipi.md

```diff
@@ -0,0 +1,181 @@
+  - vpcResourceGroup (optional string): The name of the existing VPC's resource group.
+  - subnets (optional array of strings): A list of existing subnet IDs. Leave unset and the installer will create new subnets in the VPC network on your behalf.
+  
+  **Machine Pool**
+  - type (optional string): The VSI machine profile.
+  - zones (optional array of strings): The availability zones used for machines in the pool.
+  - bootVolume (optional object):
+    - encryptionKeyCRN: (optional string): The CRN referencing a a Key Protect or Hyper Protect Crypto Services key to use for volume encryption. If not specified, a provider managed encryption key will be used.
+
+- Provide documentation (in similar format to other supported IPI platforms) to help users use and customize the IPI installer on IBM Cloud:
+  - Prerequisite instructions prior to invoking installer
+  - Description of installer options and customizations as they apply to IBM Cloud
+  - Post installation instructions on further customizations a user may wish to apply
+
+### User Stories
+
+Story 1
+As an OpenShift consumer, I want to quickly, with minimal input and default options, be able to use the OpenShift installer to create and destroy an OpenShift 4 cluster on IBM Cloud.
+
+Story 2
+As an OpenShift consumer, I want to be able to use the OpenShift installer with customizations (i.e. enhanced security) to create and destroy an OpenShift 4 cluster on IBM Cloud.
+
+Story 3
+As an OpenShift platform, I want OpenShift on IBM Cloud to be maintained and released like other supported OpenShift platforms and covered by CI tests.
+
+
+### Implementation Details/Notes/Constraints
+
+- The default IPI-provisioned cluster on IBM Cloud will be a [single region, multizone (3 zones) cluster](https://cloud.ibm.com/docs/containers?topic=containers-ha_clusters#multizone).
+
+- An IBM Cloud RHCOS image must be made available and maintained - ([current 4.7 image location](https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/4.7/latest/)). The images will be imported as custom images for IBM Cloud.
+
+- A small custom-bootstrap.ign file is used to reference canonical bootstrap.ign file hosted in object storage to workaround 64 KB user data size limit.
+
+- Customer will need to provide and prepare a [Cloud Internet Services (CIS)](https://www.ibm.com/cloud/cloud-internet-services/details) instance, which will provide required DNS capabilities.
+
+
+### Risks and Mitigations
+
+- Current RHCOS image minimum storage requirement of 120gb. Currently, IBM Cloud VSIs are provisioned with 100gb boot volumes. Workarounds have been explored and discussions have started for getting support for larger boot volume size options to accommodate image storage requirement.
+
+- The IBM Cloud Provider (Kubernetes Cloud Provider) is a work-in-progress.  It is being tracked by a [Kubernetes community enhancement](https://github.com/kubernetes/enhancements/issues/671).
```

Comment:
The external repo hasn't been created yet. It will be created in the [IBM org](https://github.com/IBM). We still have some refactoring to do on the internal repo and then flush out how we will push to upstream repo. Will update once external repo is established.